src/vm/eventtrace.cpp

```diff
@@ -1488,6 +1499,9 @@ void BulkStaticsLogger::FireBulkStaticsEvent()
 
     ULONG result = EventWrite(Microsoft_Windows_DotNETRuntimeHandle, &GCBulkRootStaticVar, _countof(eventData), eventData);
     _ASSERTE(result == ERROR_SUCCESS);
+#else
+//    _ASSERTE(!"Eventing Not Implemented");
```

Comment:
Please remove the commented out code. If its purpose was to indicate that we will need to implement this in the future, please use `// UNIXTODO: xxxx` instead.